preprocessing/tabular_data/03_training_model_on_tabular_data.ipynb

```diff
@@ -9,10 +9,7 @@
     "The purpose of this notebook is to demonstrate how to train a machine learning model via Amazon SageMaker using tabular data. In this notebook you can train either an XGBoost or Linear Learner (regression) model on tabular data in Amazon SageMaker. \n",
```

Comment:
<p>We don't need to revert the version since v2.15.</p><p>The recent version now supports v1 functions/methods with warning messages.</p><p>I think you can remove this, and in the first notebook you may want to change <code>sagemaker&gt;=2.0</code>  to <code>sagemaker&gt;=2.15.0</code>  for pip install.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>